helm_deploy/values-preprod.yaml

```diff
@@ -18,4 +18,5 @@ generic-service:
 # CloudPlatform AlertManager receiver to route promethues alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
-  alertSeverity: syscon
+  sqsAlertsQueueNames:
+    - "Digital-Prison-Services-preprod-hmpps_prisoner_to_nomis_dlq"
```

Comment:
this should have both the normal queue and also the dql here

Reply:
ok , pushed again